gammapy/datasets/spectrum.py

```diff
@@ -252,9 +252,11 @@ def peek(self, figsize=(16, 4)):
 
 
 class SpectrumDataset(PlotMixin, MapDataset):
-    """
-    Bundle together binned counts, background, IRFs, models and compute a likelihood into 1D spectrum.
-    It uses the Cash statistics by default.
+    """Main dataset for spectrum fitting (1D analysis).
+    It bundles together binned counts, background, IRFs into `~gammapy.maps.RegionNDMap` with only one spatial bin.
+    A safe mask and a fit mask can be added to exclude bins during the analysis.
+    It can contain source sky models, and in this case it can compute a likelihood for a spectral analysis.
+    It uses the Cash statistics by default (see `~gammapy.stats.cash`).
```

Comment:
```suggestion
    If models are assigned to it, it can compute the predicted number of counts and the statistic function values. 
    It uses the Cash statistic (see `~gammapy.stats.cash`).
```